configs/objectiv.json

```diff
@@ -23,6 +23,8 @@
     "text": "article p, article li, article td:last-child"
   },
   "selectors_exclude": ["footer"],
+  "js_render": true,
+  "js_wait": 2,
```

Comment:
I'd suggest to start with `1` which should be enough and increase if necessary, since this `wait` time is applied on **every** pages

Reply:
Sounds good, will update. Thanks.

Reply:
Updated